modules/media/Setup.md

```diff
@@ -0,0 +1,27 @@
+## Media module set up 
+
+Set the `mdiaPath` config value to the desired path on the server where the files 
+will be uploaded.
+
+Create the directory on the server.
+
+The upload path must be readable and writable by your web server; 
+either the web server `user` or `group` must have read and write permissions.
+
+The default group for your web server process is listed below
+```
+Ubuntu: $GROUP$ = www-data
+CentOS: $GROUP$ = apache
+```
+
+To find the `user` of your web server, run 
+`ps aux | grep 'apache' | egrep -v 'grep|Ss' | awk '{ print $1 }' | sort | uniq`
+To find the `group` of your web server, run 
+`ps aux | grep 'apache' | egrep -v 'grep|Ss' | awk '{ print $1 }' | sort | uniq | groups`
+
+To see if your web server's user or group owns the upload path, run 
+`ls -ld /data/uploads | awk '{ print "user:" $3 ", group:" $4 }'`
+
+If neither owns the folder, you should run 
+`sudo chown <unix-user>:<web-server-group> /data/uploads`
```

Comment:
Should not it be `<web-server-user>:<web-server-group>`?

Reply:
I think that would block manual changes... it also depends on the chmod and i think this is currently how permissions are on prods

Reply:
It is just which user will get permission in that case. If you have 2 admins, 1 on them will have to sudo every call unless he is member of <web-server-group>

Reply:
but if it becomes `<web-server-user>` then all users need to do sudo. and in most cases there should only be a user accessing the directory directly.